modules/simphera_aws_instance/postgresql.tf

```diff
@@ -1,3 +1,17 @@
+data "aws_subnet" "example" {
+  for_each = toset(var.private_subnets)
+  id       = each.value
+}
+
+resource "aws_db_subnet_group" "database" {

```

Comment:
I recommend to change naming to something like database_subnet_group or something similar since resource is not database but subnet group of database.

Reply:
This is aligned with previous setup (same naming etc). You can check the steps in MIGRATION.md and see the previous names of resources.